### PR TITLE
Fix arm mach o i aj format

### PR DESCRIPTION
**Pull request by xtsop**
Completed the fix by applied the "bins" : [ ... ] rule for xtr format

### CODE REVIEW

File: libr/bin/bin.c [hunk 1]
```diff
@@ -2267,6 +2267,9 @@ static void list_xtr_archs(RBin *bin, int mode) {
 		RBinXtrData *xtr_data;
 		int bits, i = 0;
 		char *arch, *machine;
+	  if (mode == 'j') {
```
Comment (radare) [inline libr/bin/bin.c]: Wrong indent

File: libr/bin/bin.c [hunk 2]
```diff
@@ -2267,6 +2267,9 @@ static void list_xtr_archs(RBin *bin, int mode) {
 		RBinXtrData *xtr_data;
 		int bits, i = 0;
 		char *arch, *machine;
```
Comment (radare) [inline libr/bin/bin.c]: Wrong indent

File: libr/bin/bin.c [hunk 3]
```diff
@@ -2298,6 +2301,9 @@ static void list_xtr_archs(RBin *bin, int mode) {
 				break;
 			}
 		}
+	  if (mode == 'j') {
```
Comment (radare) [inline libr/bin/bin.c]: Wrong indent

**Comment by Maijin** [comment]
(See CONTRIBUTING.md and sys/indent.sh script for identation)

**Comment by xtsop** [comment]
On my end the indentation is fine, the problem is created after the git push; I also run the sys/indent.sh and same thing happens.

**Comment by xtsop** [comment]
fixed the issue, vim was messing it up. You can merge now.

**Comment by radare** [comment]
Thanks!

**Pull request merged**